src/graph_notebook/neptune/client.py

```diff
@@ -68,6 +68,10 @@
 EXTRA_HEADERS = {'content-type': 'application/json'}
 SPARQL_ACTION = 'sparql'
 
+STREAM_AT = 'AT_SEQUENCE_NUMBER'
```

Comment:
Do you think it might make sense to pull all these constants out into a separate file to simplify the code?

Reply:
I did pull them out from streams.py into client.py which is where most of our constants are for now. I think in a future update creating a constants.py makes sense. I had also mentioned this to @michaelnchin